graspologic/pipeline/embed/embeddings.py

```diff
@@ -0,0 +1,122 @@
+"""
+Embeddings
+==========
+
+The ``Embeddings`` and ``EmbeddingsView`` classes combine the 2 dimensional
+embeddings generated for a graph and the associated node labels for each embedding.
+
+The embeddings generated follow the natural ordering of the provided networkx Graph
+object, but as they are positionally correlated and the source graph object may diverge
+in that natural ordering due to mutations, it is often helpful to combine a snapshot
+of the labels at the same time the embedding was genrated with the embedding to
+guarantee the appropriate latent positions are returned for each node.
+"""
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, Tuple
+import numpy as np
+from collections import OrderedDict
+
+
+class Embeddings:
```

Comment:
i will likely lose this battle but IMO a container with just an index and a numpy array of values is quite similar to a pandas dataframe, which literally has an index array and a numpy array, with the additional advantage that it is ubiquitous and easy for plotting. and if we are making someone learn our obscure data handling object vs a pandas dataframe, i'd vote for pandas.

Reply:
ubiquitous in the datascience domain, not in the application domain.  I'm trying to build for the latter, specifically.